api/routes.py

```diff
@@ -298,6 +298,12 @@ def shared_collection_hold_info(collection_name, hold_id):
 def shared_collection_revoke_hold(collection_name, hold_id):
     return app.manager.shared_collection_controller.revoke_hold(collection_name, hold_id)
 
+@library_route('/marc')
+@has_library
+@returns_problem_detail
+def marc_page():
+    return app.manager.marc_records.download_page()
+
```

Comment:
We're okay that this doesn't require authentication?

(Obviously the S3 resource doesn't require authentication but that would be easy for a library to obscure.)

Reply:
Yeah, there's no sensitive information in here. It's intended to end up in the library's public catalog anyway.